master/buildbot/data/builds.py

```diff
@@ -20,6 +20,31 @@
 
 class Db2DataMixin(object):
 
+    def _generate_filtered_properties(self, props, filters):
+        """
+        This method returns Build's properties according to property filters.
```

Comment:
This part should be rather documented in the data api section of the doc. (you can add a reference to it here. dont duplicate doc.)